zip-0209.rst

```diff
@@ -0,0 +1,48 @@
+::
+
+  ZIP: 209
+  Title: Prohibit Negative Shielded Value Pool
+  Author: Sean Bowe <sean@z.cash>
+  Category: Consensus
+  Created: 2019-02-25
+  License: MIT
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The "Sprout value pool balance", as implied by section 4.11 of the Zcash Protocol Specification [#protocol]_, is the sum of all previous transactions' ``vpub_old`` fields subtracted by the sum of all previous transactions ``vpub_new`` fields.
```

Comment:
Nit: the second "transactions" should also be possessive.

Reply:
Is "previous" well-defined here? Maybe

> The term "block chain" and "network upgrade" are to be interpreted as defined in [#zip-0200]_.
>
> The "Sprout value pool balance" for a given block chain, as implied by section 4.11 of the Zcash Protocol Specification [#protocol]_, is the sum of all ``vpub_old`` fields for transactions in the block chain, minus the sum of all ``vpub_new`` fields for transactions in the block chain.
>
> The "Sapling value pool balance" for a given block chain, as implied by section 4.12 of Zcash Protocol Specification [#protocol]_, is the negation of the sum of all ``valueBalance`` fields for transactions in the block chain.

Reply:
Agreed that the word previous is not necessary unless we are defining the value pools at a specific point in time. I also prefer 'minus' to 'subtracted by'.